<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>645</width>
    <height>786</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Note categories</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="note_categories_tab">
        <attribute name="title">
         <string notr="true">Cat√©gories</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="1" column="1">
          <layout class="QGridLayout" name="gridLayout_7">
           <item row="1" column="2">
            <widget class="QPushButton" name="hide_note_category_button">
             <property name="sizePolicy">
              <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string notr="true">Cacher &gt;</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <spacer name="verticalSpacer_4">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Minimum</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="0" rowspan="4" colspan="2">
            <widget class="QListWidget" name="shown_note_categories_list">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="selectionMode">
              <enum>QAbstractItemView::ExtendedSelection</enum>
             </property>
             <property name="sortingEnabled">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QPushButton" name="show_note_category_button">
             <property name="sizePolicy">
              <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string notr="true">&lt; Montrer</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0" colspan="2">
            <widget class="QLineEdit" name="add_note_category_input">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QPushButton" name="add_note_category_button">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string notr="true">Ajouter</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QPushButton" name="delete_note_category_button">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string notr="true">Supprimer</string>
             </property>
            </widget>
           </item>
           <item row="0" column="3" rowspan="6">
            <widget class="QListWidget" name="hidden_note_categories_list">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="selectionMode">
              <enum>QAbstractItemView::ExtendedSelection</enum>
             </property>
             <property name="sortingEnabled">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="3" column="2" rowspan="3">
            <spacer name="verticalSpacer_3">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Minimum</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="notes_tab">
        <attribute name="title">
         <string notr="true">Notes</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_6">
         <item row="1" column="0">
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="0" column="1">
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="3" column="1">
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="1" column="1">
            <widget class="QPushButton" name="add_notes_in_category_btn">
             <property name="text">
              <string notr="true">Ajouter &gt;&gt;&gt;</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QPushButton" name="remove_notes_from_category_btn">
             <property name="text">
              <string notr="true">&lt;&lt;&lt; Enlever</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0" rowspan="4">
            <widget class="FilteredNotesListWidget" name="notes_not_in_category_list" native="true">
             <property name="sizePolicy">
              <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item row="0" column="2" rowspan="4">
            <widget class="FilteredNotesListWidget" name="notes_in_category_list" native="true">
             <property name="sizePolicy">
              <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="0">
          <widget class="QComboBox" name="note_categories_selector"/>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FilteredNotesListWidget</class>
   <extends>QWidget</extends>
   <header>gui/filtered_notes_list_widget</header>
   <container>1</container>
   <slots>
    <signal>itemSelectionChanged()</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>add_note_category_input</sender>
   <signal>returnPressed()</signal>
   <receiver>Dialog</receiver>
   <slot>add_note_category_fnc()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>191</x>
     <y>756</y>
    </hint>
    <hint type="destinationlabel">
     <x>409</x>
     <y>392</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>add_note_category_button</sender>
   <signal>pressed()</signal>
   <receiver>Dialog</receiver>
   <slot>add_note_category_fnc()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>409</x>
     <y>756</y>
    </hint>
    <hint type="destinationlabel">
     <x>409</x>
     <y>392</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>show_note_category_button</sender>
   <signal>pressed()</signal>
   <receiver>Dialog</receiver>
   <slot>show_note_category_fnc()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>409</x>
     <y>398</y>
    </hint>
    <hint type="destinationlabel">
     <x>409</x>
     <y>392</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hide_note_category_button</sender>
   <signal>pressed()</signal>
   <receiver>Dialog</receiver>
   <slot>hide_note_category_fnc()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>409</x>
     <y>378</y>
    </hint>
    <hint type="destinationlabel">
     <x>409</x>
     <y>392</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>add_notes_in_category_btn</sender>
   <signal>pressed()</signal>
   <receiver>Dialog</receiver>
   <slot>add_notes_in_category_fnc()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>409</x>
     <y>404</y>
    </hint>
    <hint type="destinationlabel">
     <x>409</x>
     <y>392</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>note_categories_selector</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>Dialog</receiver>
   <slot>dropdown_item_changed_fnc()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>409</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>409</x>
     <y>392</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>remove_notes_from_category_btn</sender>
   <signal>pressed()</signal>
   <receiver>Dialog</receiver>
   <slot>remove_notes_from_category_fnc()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>409</x>
     <y>424</y>
    </hint>
    <hint type="destinationlabel">
     <x>409</x>
     <y>392</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>delete_note_category_button</sender>
   <signal>pressed()</signal>
   <receiver>Dialog</receiver>
   <slot>delete_note_category_fnc()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>210</x>
     <y>759</y>
    </hint>
    <hint type="destinationlabel">
     <x>322</x>
     <y>392</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>shown_note_categories_list</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>Dialog</receiver>
   <slot>on_shown_note_category_change(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>145</x>
     <y>378</y>
    </hint>
    <hint type="destinationlabel">
     <x>322</x>
     <y>392</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hidden_note_categories_list</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>Dialog</receiver>
   <slot>on_hidden_note_category_change(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>498</x>
     <y>401</y>
    </hint>
    <hint type="destinationlabel">
     <x>322</x>
     <y>392</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>add_note_category_fnc()</slot>
  <slot>hide_note_category_fnc()</slot>
  <slot>show_note_category_fnc()</slot>
  <slot>add_notes_in_category_fnc()</slot>
  <slot>remove_notes_from_category_fnc()</slot>
  <slot>dropdown_item_changed_fnc()</slot>
  <slot>delete_note_category_fnc()</slot>
  <slot>on_shown_note_category_change(QString)</slot>
  <slot>on_hidden_note_category_change(QString)</slot>
 </slots>
</ui>
